Problem2.java

```diff
@@ -0,0 +1,42 @@
+// Time Complexity : O(N), N is size of string s + sum of nums in s
+// Space Complexity : O(N)
+// Did this code successfully run on Leetcode : Y
+// Any problem you faced while coding this :
+
+
+// Your code here along with comments explaining your approach
+class Solution {
+    public String decodeString(String s) {
```

Comment:
Good job! Please add comments explaining your approach. Everything else seems fine.

Reply:
@yahska-ednihs Code is already commented. Are you expecting more comments?